templates/.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
VSCode settings to ignore showing files in the file explorer and the search results.